cfgov/jinja2/v1/youth_employment_success/route-questions/driving-cost-estimate.html

```diff
@@ -1,11 +1,18 @@
 <div class="content-l block block__sub js-driving-estimate">
   <div class="content-l_col content-l_col-1 a-label">
     <p class="a-label__heading u-mb0">
-      Average cost per mile
+      Average daily cost
     </p>
     <small>
-      $0.80 per mile is an estimated number that includes more than just gas &mdash; it also factors in average maintenance costs, insurance, and loan payments, based on data from AAA.
+      We've calculated this estimated number for you based on data from <a href="https://www.bts.gov/content/average-cost-owning-and-operating-automobile" rel="noopener noreferrer" target="_blank">Department of Transportation, Bureau of Transportation Statistics</a> about average cost per mile, which includes 
+      <ul>
+      	<li>gas</li>
+      	<li>insurance</li>
+      	<li>maintenance costs</li>
+      	<li>tires</li>
+      	<li>registration and taxes</li>
+      	<li>depreciation,</li>
+      	<li>finance charges.</li>
```

Comment:
Missing closing `</ul>` looks like.

Reply:
Thanks for catching that! Fixed in https://github.com/cfpb/cfgov-refresh/pull/5780/commits/843b349e9a8ba91c1f5cebd2c165ea558305f751.